### PR TITLE
Snapshot captured at deletable

### PR DESCRIPTION
**Pull request by BShrivastav**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000
https://jiracloud.swg.usma.ibm.com:8443/browse/UI-19113

Output from acceptance testing:

```
$ make testacc TEST=./ibm/service/vpc TESTARGS='-run=TestAccIBMISSnapshotDatasource_basic'
--- PASS: TestAccIBMISSnapshotDatasource_basic (363.04s)
PASS
ok  	github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/vpc	364.028s


$ make testacc TEST=./ibm/service/vpc TESTARGS='-run=TestAccIBMISSnapshotsDatasource_basic'

--- PASS: TestAccIBMISSnapshotsDatasource_basic (387.87s)
PASS
ok  	github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/vpc	389.266s

...
```

### CODE REVIEW

File: ibm/service/vpc/data_source_ibm_is_snapshot_test.go [hunk 1]
```diff
@@ -20,8 +20,11 @@ func TestAccIBMISSnapshotDatasource_basic(t *testing.T) {
...
+	// `)
 	publicKey := strings.TrimSpace(`
-ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCKVmnMOlHKcZK8tpt3MP1lqOLAcqcJzhsvJcjscgVERRN7/9484SOBJ3HSKxxNG5JN8owAjy5f9yYwcUg+JaUVuytn5Pv3aeYROHGGg+5G346xaq3DAwX6Y5ykr2fvjObgncQBnuU5KHWCECO/4h8uWuwh/kfniXPVjFToc+gnkqA+3RKpAecZhFXwfalQ9mMuYGFxn+fwn8cYEApsJbsEmb0iJwPiZ5hjFC8wREuiTlhPHDgkBLOiycd20op2nXzDbHfCHInquEe/gYxEitALONxm0swBOwJZwlTDOB7C6y2dzlrtxr1L59m7pCkWI4EtTRLvleehBoj3u7jB4usR
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDCZNT0p7DKLKoXshjO0xHuPVpxAHRYhTkVI+yklxPoF/1UuIy+8cczEUnEoIZ81hHVnyY8Ltr75Xr2/yI1g+a6t+xwgdp74f5+R69AuIuywsoLdt8mda2iFryrPs5eebhSBo9c2dGya/uv1pR8o/ED9xrIB4QQ+vrT8uuss3GIP6lR5w9vJqWgeB4I4TqGu4giWLkeMyBKVVUn/UoAqoa7uD7deFXkmN73j227AyCV+lx8MBFQZKvnsglJxgSnjwlQP2iWCyLGvDV1j0lldXAvjkxYRl8H9ilrOk5KrCpqVtlsXcXqqBTeCEBpI0KOHIAl/JJ+YFc3+YTQq0waGB82CaqQCnJEy6bBpdPz4tge16/2BehY0fY5AG8sqb7Or91ilsE0nnuR3Tzk5l9YS8ftqyjYAoRbBy3P7AF433qbCgoclndp5Mcy5EsEJ//kme0u4n8ZF20ZS8oqKK1QIfOio8U2er/SSF3UAHhT2n/hKaeGeXv0kFwCsExjP1iZsHk= bhaveshshrivastav@Bhaveshs-MacBook-Pro.local
```
Comment (hkantare) [inline ibm/service/vpc/data_source_ibm_is_snapshot_test.go:27]: can you remove u r ssh key

Comment (BShrivastav) [reply ibm/service/vpc/data_source_ibm_is_snapshot_test.go]: done.

File: ibm/service/vpc/data_source_ibm_is_snapshots_test.go [hunk 2]
```diff
@@ -20,9 +20,7 @@ func TestAccIBMISSnapshotsDatasource_basic(t *testing.T) {
 	vpcname := fmt.Sprintf("tf-vpc-%d", acctest.RandIntRange(10, 100))
 	name := fmt.Sprintf("tf-instnace-%d", acctest.RandIntRange(10, 100))
 	subnetname := fmt.Sprintf("tf-subnet-%d", acctest.RandIntRange(10, 100))
-	publicKey := strings.TrimSpace(`
-ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCKVmnMOlHKcZK8tpt3MP1lqOLAcqcJzhsvJcjscgVERRN7/9484SOBJ3HSKxxNG5JN8owAjy5f9yYwcUg+JaUVuytn5Pv3aeYROHGGg+5G346xaq3DAwX6Y5ykr2fvjObgncQBnuU5KHWCECO/4h8uWuwh/kfniXPVjFToc+gnkqA+3RKpAecZhFXwfalQ9mMuYGFxn+fwn8cYEApsJbsEmb0iJwPiZ5hjFC8wREuiTlhPHDgkBLOiycd20op2nXzDbHfCHInquEe/gYxEitALONxm0swBOwJZwlTDOB7C6y2dzlrtxr1L59m7pCkWI4EtTRLvleehBoj3u7jB4usR
-`)
+	publicKey := strings.TrimSpace(`yourkey`)
```
Comment (hkantare) [inline ibm/service/vpc/data_source_ibm_is_snapshots_test.go:23]: where u defined this yourkey?

Comment (BShrivastav) [reply ibm/service/vpc/data_source_ibm_is_snapshots_test.go]: I have reverted the key to be used as referenced in the other cases.

**Pull request merged**